.Rbuildignore

```diff
@@ -8,3 +8,4 @@ $credentials.*
 $database.*
 ^README\.Rmd$
 ^\.github$
+^LICENSE\.md$
```

Comment:
It feels like we should add some items to `.Rbuildignore` to knock out a couple notes.

![image](https://user-images.githubusercontent.com/85879620/202541980-bffb2005-a4ef-42ff-a721-97717bbb3ad5.png)

Reply:
Yeah I think I can find which functions from `crayon`, `desc` and `forcats` are being used here.
We're getting rid of `www`
And I can add some files to .gitignore